kafka/pom.xml

```diff
@@ -7,117 +7,67 @@
   OF ANY KIND, either express or implied. See the License for the specific language
   governing permissions and limitations under the License. -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>io.cloudevents</groupId>
-    <artifactId>cloudevents-parent</artifactId>
-      <version>2.0.0-SNAPSHOT</version>
-  </parent>
+    <parent>
+        <groupId>io.cloudevents</groupId>
+        <artifactId>cloudevents-parent</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
 
-  <groupId>io.cloudevents</groupId>
-  <artifactId>cloudevents-kafka</artifactId>
-    <version>${project.parent.version}</version>
-  <packaging>jar</packaging>
+    <artifactId>cloudevents-kafka</artifactId>
+    <name>CloudEvents - Kafka Binding</name>
+    <packaging>jar</packaging>
 
-  <name>CloudEvents - Kafka Transport Binding</name>
+    <properties>
+        <kafka.version>2.5.0</kafka.version>
+    </properties>
 
-  <properties>
-    <jackson.version>2.10.1</jackson.version>
-    <junit-jupiter.version>5.3.2</junit-jupiter.version>
-    <kafka.version>2.0.1</kafka.version>
-  </properties>
+    <dependencies>
+        <dependency>
+            <groupId>io.cloudevents</groupId>
+            <artifactId>cloudevents-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
```

Comment:
provided scope?

Reply:
Mh I would prefer to have it compile with compile scope